qemu/tests/hotplug_mem_reserve.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import time
+
+from virttest.utils_misc import normalize_data_size
+from virttest.staging import utils_memory
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+def run(test, params, env):
+    """
+    Qemu memory hotplug test:
+    1) Boot guest with -m option
+    2) Hotplug memory to guest with option reserve enable/disabled
+    3) Check memory inside guest
+    3) Check hugepages on host
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.wait_for_login()
+    mem_name = params["target_mems"]
+    hotplug_test = MemoryHotplugTest(test, params, env)
+    hotplug_test.hotplug_memory(vm, mem_name)
+    hotplug_test.check_memory(vm)
+    time.sleep(30)
```

Comment:
I wonder if there is any effective method to check it is ready or not ?

Reply:
Maybe use wait_for to check HugePages_Rsvd every 5 seconds untils it's stable?

Reply:
Agree with that